htmd/adaptive/adaptiverun.py

```diff
@@ -276,29 +276,41 @@ def __setattr__(self, key, value):
         uisetattr(self, key, value, all)
 
     def _algorithm(self):
+
         logger.info('Postprocessing new data')
-        datalist = simlist(glob(path.join(self.datapath, '*', '')), glob(path.join(self.inputpath, '*', 'structure.pdb')),
-                           glob(path.join(self.inputpath, '*', '')))
+
+        if self.app.get_class() == 'PmemdLocal':
+            datalist = simlist(glob(path.join(self.datapath, '*', '')),
+                               glob(path.join(self.inputpath, '*', 'structure.prmtop')),
+                               glob(path.join(self.inputpath, '*', '')))
+        else:
+            datalist = simlist(glob(path.join(self.datapath, '*', '')),
+                               glob(path.join(self.inputpath, '*', 'structure.pdb')),
+                               glob(path.join(self.inputpath, '*', '')))
         filtlist = simfilter(datalist, self.filteredpath, filtersel=self.filtersel)
 
+        # calculate metrics for tica projection
         if hasattr(self, 'metricsel2') and self.metricsel2 is not None:
```

Comment:
I apologize for the wide lack of comments in the code :( Will improve in the future.